package.json

```diff
@@ -35,6 +35,7 @@
     "identity-obj-proxy": "^3.0.0",
     "jest": "^29.5.0",
     "jest-environment-jsdom": "^29.5.0",
+    "jsdom": "^22.0.0",
```

Comment:
This library is not helpful in our project, we are not using it. It may be better to remove it from this file.